tasks/xgettext.js

```diff
@@ -319,48 +370,55 @@ module.exports = function(grunt) {
 
       var messages = {};
       f.src.forEach(function(file) {
```

Comment:
Mixing native forEach and _.each is misleading as to the specific intent.

Reply:
yeah, definitely planning on eliminating underscore as a dependency